src/main/java/io/aiven/kafka/connect/common/grouper/RecordGrouperFactory.java

```diff
@@ -16,6 +16,7 @@
 
 package io.aiven.kafka.connect.common.grouper;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.LinkedHashMap;
 import java.util.List;
```

Comment:
```suggestion
```

Reply:
`List` is still used on line 41 (and probably others).